docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc

```diff
@@ -163,7 +163,7 @@ NAME                             URL
 080dbba8b3ffba35739ebe5bce69a2be https://github.com/apache/commons-logging.git    LOGGING_1_2        DependencyBuildStateComplete
 ----
 
-The `PipelineRun` objects associated with these builds start with the build name, so you can view the logs for a build under the appropriate `PipelineRun`.
+The `PipelineRun` objects associated with these builds start with the build name, so you can view the logs for a build for the appropriate `PipelineRun`.
```

Comment:
I have a few changes -- I know you didn't have time for style so some of that might be overlapping:

The names of the PipelineRun objects begin with the build name. This enables you to view logs for each PipelineRun.